custom_components/powercalc/data/innr/RB 148 T/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Nous A1T",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.1:docker"
+    },
+    "name": "INNR",
```

Name needs to be more descriptive about the light bulb. Maybe something like this.
Will be displayed in `name` column of [support model list](https://github.com/bramstroker/homeassistant-powercalc/blob/master/docs/supported_models.md#Lights)

```suggestion
    "name": "RB 148 T - E14 Tunable White",
```